tests/SummitOrderServiceTest.php

```diff
@@ -46,4 +53,29 @@ public function testProcessSummitOrderReminders() {
         $service = App::make(ISummitOrderService::class);
         $service->processSummitOrderReminders(self::$summit);
     }
+
+    public function testAutoAssignPrePaidTicket() {
```

Comment:
we need to add a test for scenario on where there are not more available tickets on the promo code